Dockerfile

```
@@ -68,13 +65,13 @@ RUN addgroup \
     git && \
   echo "git:*" | chpasswd -e
 
+COPY --from=build-env /tmp/local /
+COPY --chmod=755 --from=build-env /go/src/code.gitea.io/gitea/gitea /app/gitea/gitea
```

`--chmod=755` is not needed because there is a complete `chmod 755` above


```suggestion
COPY --from=build-env /go/src/code.gitea.io/gitea/gitea /app/gitea/gitea
```